controller_manager:
  ros__parameters:
    update_rate: 100
    joint_state_broadcaster:
      type: joint_state_broadcaster/JointStateBroadcaster
    camera_pan_position_controller:
      type: forward_command_controller/ForwardCommandController

camera_pan_position_controller:
  ros__parameters:
    joints: ["camera_pan_joint"]
    interface_name: position
